experimental/add2app/SimpleIOSExample/SimpleIOSExample/ViewController.swift

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import UIKit
+import Flutter
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var launchButton: UIButton!
```

Comment:
I'd just not connect it if you're not gonna use it

Reply:
Fair enough. Removed.